StreamChatUITestsAppUITests/MockServer/ChannelResponses.swift

```diff
@@ -10,10 +10,19 @@ extension StreamMockServer {
     // TODO: CIS-1686
     func configureChannelEndpoints() {
         server[MockEndpoint.query] = { _ in
-            .ok(.text(TestData.getMockResponse(fromFile: .httpChannel)))
+            self.mockChannel(mockFile: .httpChannels)
```

Comment:
`weak`ify self here.

Reply:
Do you think it's required? If it's weak, how could we unwrap it?

Reply:
@alteral you could return `empty` value
```
self?.mockChannel(mockFile: .httpChannels) ?? .badRequest(nil)
```

or leverage other strategies when weakifying `self`.

Lets avoid memory issues.